css-typed-om/EXPLAINER.md

```diff
@@ -0,0 +1,117 @@
+# CSS Typed OM Explained
+
+CSS Typed OM is the extensable API for the CSSOM that reduces the burden of manipulating a CSS propeerty's value via string manipulation. It does so by exposing CSS values as typed JavaScript objects which help you circumvent the browser's CSS parser.
```

Comment:
s/extensable/extensible

Reply:
s/propeerty/property

Reply:
not just the browser's CSS parser, but also JavaScript CSS parsers. Maybe something like "typed JavaScript objects to avoid string parsing", or even just "typed JavaScript objects rather than strings".

Reply:
Done * 3